### PR TITLE
#18571 Testing

### PR DESCRIPTION
**Pull request by freddyucv**
Testing https://github.com/dotCMS/enterprise-2.x/pull/739

### CODE REVIEW

**Review by jdotcms — APPROVED** [review]

**Review by erickgonzalez — APPROVED** [review]

File: dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.CacheLocator;
+import com.dotmarketing.business.DotCacheException;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.rules.model.Condition;
+import com.dotmarketing.portlets.rules.model.ConditionGroup;
+import com.dotmarketing.portlets.structure.model.ContentletRelationships;
+import com.liferay.portal.model.User;
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java:13]: Codacy found an issue: [Avoid unused imports such as 'com.liferay.portal.model.User'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5889283)

File: dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        final File createRelationshipContentBundle = new File(getClass().getResource(
+                "/bundle/relationship_push_publish/create_relationship_content.tar.gz")
+                .getFile());
+
+        final File removeRelationshipContentBundle = new File(getClass().getResource(
+                "/bundle/relationship_push_publish/remove_relationship_in_content.tar.gz")
+                .getFile());
+
+        final FileInputStream createCOntentFileInputStream = new FileInputStream(createRelationshipContentBundle);
+        final FileInputStream removeRelationshipFileInputStream = new FileInputStream(removeRelationshipContentBundle);
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java:60]: Codacy found an issue: [Ensure that resources like this FileInputStream object are closed after use](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5889283)

File: dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+import com.dotcms.publisher.business.DotPublisherException;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.CacheLocator;
+import com.dotmarketing.business.DotCacheException;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.rules.model.Condition;
+import com.dotmarketing.portlets.rules.model.ConditionGroup;
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java:11]: Codacy found an issue: [Avoid unused imports such as 'com.dotmarketing.portlets.rules.model.ConditionGroup'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5889283)

File: dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+        final File createRelationshipContentBundle = new File(getClass().getResource(
+                "/bundle/relationship_push_publish/create_relationship_content.tar.gz")
+                .getFile());
+
+        final File removeRelationshipContentBundle = new File(getClass().getResource(
+                "/bundle/relationship_push_publish/remove_relationship_in_content.tar.gz")
+                .getFile());
+
+        final FileInputStream createCOntentFileInputStream = new FileInputStream(createRelationshipContentBundle);
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java:59]: Codacy found an issue: [Ensure that resources like this FileInputStream object are closed after use](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5889283)

File: dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java [hunk 5]
```diff
@@ -0,0 +1,94 @@
+package com.dotcms.rest;
+
+import com.dotcms.publisher.business.DotPublisherException;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.CacheLocator;
+import com.dotmarketing.business.DotCacheException;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.rules.model.Condition;
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java:10]: Codacy found an issue: [Avoid unused imports such as 'com.dotmarketing.portlets.rules.model.Condition'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5889283)

File: dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+
+import static com.dotcms.util.CollectionsUtils.list;
+import static com.dotcms.util.CollectionsUtils.toImmutableList;
+import static org.junit.Assert.*;
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java:29]: Codacy found an issue: [Avoid unused imports such as 'org.junit.Assert'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5889283)

File: dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+import com.dotcms.publisher.business.DotPublisherException;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.CacheLocator;
+import com.dotmarketing.business.DotCacheException;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.rules.model.Condition;
+import com.dotmarketing.portlets.rules.model.ConditionGroup;
+import com.dotmarketing.portlets.structure.model.ContentletRelationships;
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java:12]: Codacy found an issue: [Avoid unused imports such as 'com.dotmarketing.portlets.structure.model.ContentletRelationships'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5889283)

File: dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+import org.junit.Test;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+
+import static com.dotcms.util.CollectionsUtils.list;
+import static com.dotcms.util.CollectionsUtils.toImmutableList;
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java:28]: Codacy found an issue: [Avoid unused imports such as 'com.dotcms.util.CollectionsUtils.toImmutableList'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5889283)

File: dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class BundleResourceTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java:36]: Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5889283)

File: dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java (hunk 4 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java:59]: Codacy found an issue: [Avoid instantiating FileInputStream, FileOutputStream, FileReader, or FileWriter](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5889283)

File: dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java (hunk 2 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java:60]: Codacy found an issue: [Avoid instantiating FileInputStream, FileOutputStream, FileReader, or FileWriter](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5889283)

File: dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java (hunk 7 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java:12]: Codacy found an issue: [Avoid unused imports such as 'com.dotmarketing.portlets.structure.model.ContentletRelationships'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5889283)

File: dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java (hunk 6 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java:29]: Codacy found an issue: [Avoid unused imports such as 'org.junit.Assert'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5889283)

File: dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java (hunk 4 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java:59]: Codacy found an issue: [Avoid instantiating FileInputStream, FileOutputStream, FileReader, or FileWriter](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5889283)

File: dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java (hunk 5 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java:10]: Codacy found an issue: [Avoid unused imports such as 'com.dotmarketing.portlets.rules.model.Condition'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5889283)

File: dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java (hunk 9 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java:36]: Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5889283)

File: dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java (hunk 2 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java:60]: Codacy found an issue: [Ensure that resources like this FileInputStream object are closed after use](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5889283)

File: dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java (hunk 1 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java:13]: Codacy found an issue: [Avoid unused imports such as 'com.liferay.portal.model.User'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5889283)

File: dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java (hunk 8 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java:28]: Codacy found an issue: [Avoid unused imports such as 'com.dotcms.util.CollectionsUtils.toImmutableList'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5889283)

File: dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java (hunk 4 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java:59]: Codacy found an issue: [Ensure that resources like this FileInputStream object are closed after use](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5889283)

File: dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java (hunk 2 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java:60]: Codacy found an issue: [Avoid instantiating FileInputStream, FileOutputStream, FileReader, or FileWriter](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5889283)

File: dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java (hunk 3 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java:11]: Codacy found an issue: [Avoid unused imports such as 'com.dotmarketing.portlets.rules.model.ConditionGroup'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5889283)

**Pull request merged**